config/locales/en.yml

```diff
@@ -11,6 +11,15 @@ en:
       scitt: SCITT
       university: University
       lead_school: Lead partner
+    school_options_form_component:
+      add_organisation: Add organisation
+      continue: Continue
+      hint_many_schools_html: Showing the first %{school_count} results. %{link_to} if the school you’re looking for is not listed.
+      hint_html: '%{link_to} if the school you’re looking for is not listed.'
+      hint_link: Change your search
+      hint_many_schools_link: Try narrowing down your search
```

Comment:
This is more of a question - not a strong opinion or change request.

It occurs to me that we have a couple of approaches to naming locale keys.

Some locale keys _mirror the string they represent_. For example:

https://github.com/DFE-Digital/itt-mentor-services/blob/083ba33336ddaad4de46f15daa3e18d1b6bab666/config/locales/claims/en/support/schools.yml#L13-L17

In other cases, such as here, the key _does not reflect the string value_. There's no relation between the two. It means that the ERB view is a bit more opaque, because it's not easy to guess what the localised string is.

For example, I have no idea what these lines will output – unless I go and look up the value, or have a fancy IDE configured to show the translations as hints:

https://github.com/DFE-Digital/itt-mentor-services/blob/083ba33336ddaad4de46f15daa3e18d1b6bab666/app/components/school_options_form_component.html.erb#L48-L50

To some extent I assume all apps will have a mixture of both. Not least because Active Record has a specific key format it uses for attribute names and error messages.

Is there any best practice or guidance for which naming approach is best? Or does this seem like a non-issue / I'm just being pedantic?

Reply:
This is an issue, I guess some translations don't have actions behind them l `continue`. I'll rename them